localstripe/resources.py

```diff
@@ -1415,7 +1420,7 @@ def _update(self, metadata=None, items=None, tax_percent=None,
         # If the subscription is updated to a more expensive plan, an invoice
         # is not automatically generated. To achieve that, an invoice has to
         # be manually created using the POST /invoices route.
-        self._set_up_subscription_and_invoice(plan, create_an_invoice=False)
+        self._set_up_subscription_and_invoice(plan, quantity)
```

Comment:
- Why do you remove `create_an_invoice=`? It is not consistent with the comment above.
- `quantity` is not defined if `items` is `None` or `[]`, which will lead to a crash.
- Do you mean that an invoice should be created only if `quantity` ≠ 0? In this case, maybe this would be clearer: `create_an_invoice = quantity > 0; _set_up_subscription_and_invoice(plan, create_an_invoice=create_an_invoice)`?
  If it's not what you mean, can you explain?

Reply:
You're absolutely right, I shouldn't have removed `create_an_invoice`, good catch! I'll update the PR to fix this